explore/matrix-io/01.save-matrices.ipynb

```diff
@@ -18,11 +18,17 @@
    "cell_type": "code",
    "execution_count": 2,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/bin/sh: 1: {sys.executable}: not found\r\n"
+     ]
+    }
+   ],
    "source": [
-    "import sys\n",
-    "sys.path.insert(0,'../../')\n",
-    "from hetmech.degree_weight import dwpc"
+    "!{sys.executable} -m pip install git+https://github.com/dhimmel/hetio@aa5d12ec43d2dfd349893df9efd9ef8f665c982b --upgrade"
```

Comment:
Hmm. This is not the right place to upgrade hetio.

Reply:
Right. I was not sure if I should update the `environment.yml` file or if a new hetio release was planned.

Reply:
Let's update `environment.yml` to use a GitHub commit hash rather than PyPI version. However, this PR doesn't require hetio commits past v0.2.8, so do we want to make the environment change when we need it for the next PR?

Reply:
Ah right. I forgot that I had only added that to a different notebook because I was needing the function `extract_all_metapaths`, but we are not using that here.